### PR TITLE
Database sync + .gitignore update

### PR DESCRIPTION
**Pull request by ksamson2**
Adding base changes needed for gql and changes needed for gitignore to fix amplify continuity errors

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,6 +22,4 @@ amplify-gradle-config.json
 amplifytools.xcconfig
 .secret-*
 **.sample
-#amplify-do-not-edit-end
-
-amplify/
+#amplify-do-not-edit-end
```
Comment (christoff-linde) [inline .gitignore:27]: We should add `.DS_Store` to the `.gitignore`

File: amplify/backend/api/viaweb/cli-inputs.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+  "version": 1,
+  "serviceConfiguration": {
+    "apiName": "viaweb",
+    "serviceName": "AppSync",
+    "gqlSchemaPath": "/Users/camsamso/Documents/Personal/UP/IMY 772/temp/via-web/amplify/backend/api/viaweb/schema.graphql",
```
Comment (christoff-linde) [inline amplify/backend/api/viaweb/cli-inputs.json:6]: would this absolute path not cause issues? can we update it to be a relative one

**Review by christoff-linde — CHANGES_REQUESTED** [review]

File: amplify/backend/api/viaweb/cli-inputs.json (hunk 2 above)
Comment (ksamson2) [reply amplify/backend/api/viaweb/cli-inputs.json:6]: good point.. it's an auto-generated file though so not sure how changing it works, I can take a look

Comment (ksamson2) [reply amplify/backend/api/viaweb/cli-inputs.json:6]: Not sure why this is still committing :/ I added it to the gitignore. It should be a file created by amplify

**Review by christoff-linde — APPROVED** [review]

**Pull request merged**